data/scripts/Xsession

```diff
@@ -52,6 +52,9 @@ case $SHELL in
 esac
 
 [ -f /etc/xprofile ] && . /etc/xprofile
+if [ `uname` = "FreeBSD" ]; then
```

Comment:
Is this worth checking FreeBSD-ness for? The other lines are all "does this file exist? if yes, source it" which would make sense here as well; if you have an `/usr/local/etc/profile` and you're on Linux (or Solaris) then that's probably a profile to use there as well.